.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+updates:
+- package-ecosystem: pip
+  directory: "/"
+  target-branch: dev
+  commit-message:
+    prefix: "[Dependabot]"
+  labels:
+      - Dependencies
+  assignees:
+    - Paebbels
+  reviewers:
+    - Paebbels
+  schedule:
+    interval: daily
```

Comment:
I typically prefer to set it weekly, in order to reduce the traffic/noise in the repos. That way, you get all the PRs created the same day at the same time.